### PR TITLE
MM-30706: Skip racy test TestWebSocket

### PR DESCRIPTION
**Pull request by agnivade**
We just skip the test for now

https://mattermost.atlassian.net/browse/MM-30706

```release-note
NONE
```

### CODE REVIEW

File: api4/websocket_norace_test.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// +build !race
```
Comment (streamer45) [inline api4/websocket_norace_test.go:1]: The linter didn't like this.

**Review by streamer45 — APPROVED** [review]
Looks good :+1:

**Review by isacikgoz — APPROVED** [review]

File: api4/websocket_norace_test.go (hunk 1 above)
Comment (agnivade) [reply api4/websocket_norace_test.go:1]: Ah, now I have to fix the linter.

**Pull request merged**